apps/codeforafrica/src/components/StayInTouch/StayInTouch.js

```diff
@@ -0,0 +1,96 @@
+import { Link } from "@commons-ui/next";
+import { Grid } from "@mui/material";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import * as React from "react";
+
+import Title from "./Title";
+
+const LinksRoot = styled(Grid, {
+  slot: "Root",
+  name: "StayInTouchLinks",
+})(({ theme }) => ({
+  "& > a": {
+    display: "inline-block",
+  },
+  justifyContent: "center",
+  [theme.breakpoints.up("md")]: {
+    justifyContent: "flex-start",
+    width: "auto",
+  },
```

Comment:
All of this can be in `sx` prop on `Grid item`... Feels "foreign" to see `LinksRoot item`